task30/src/test/java/by/tut/webpages/myTabel.java

```diff
@@ -0,0 +1,13 @@
+package by.tut.webpages;
+
+public class myTabel {
```

Comment:
Rename, f.e. EmployeeModel (do not use my example)

Reply:
fixed.

Changed to "EmployeeTable"